### DISCUSSION TITLE
Customize bt.plot() graph

### DISCUSSION THREAD

**Discussion by huytradingz** [kernc/backtesting.py#1076] [Q&A] [answered]
Hi all, I would like to ask if we could further customize the plot bt.plot() return. For example,  I am writing a breakout strategy and would like to draw support and resistance lines on bt.plot() graph. Do I need to edit the plot functions in the source or there are simpler ways? Thank you!

**Comment by lapitskaya38** [comment, marked as answer]
Hello. Are these regular horizontal lines? Are they not in the Talib?? If not, then make two indicators, a numpy array with the calculation of values for constructing lines. And add to I.

`self.my_line1 = self.I(lambda: self.my_numpy1, overlay=True)
self.my_line2 = self.I(lambda: self.my_numpy2, overlay=True)`

**Reply by huytradingz** [reply]
Sorry for the delayed response and thank you for your suggestion! Will try it out and update!
